src/Controls/tests/TestCases.HostApp/Issues/Issue29325.cs

```diff
@@ -0,0 +1,29 @@
+namespace Maui.Controls.Sample.Issues;
+[Issue(IssueTracker.Github, 29325, "Button Shadow Color Transparency Not Applied Correctly", PlatformAffected.Android)]
+public class Issue29325 : ContentPage
+{
+	public Issue29325()
+	{
+		var verticalStackLayout = new VerticalStackLayout();
```

Comment:
Can you make this test a bit more full?

I am thinking that maybe you have tests with:
* no alpha nor opacity
* alpha
* opacity
* alpha and opacity.

And theoretically it should appear that the alpha and the opacity are the same, but the alpha AND opacity is much less visible.